src/services/API/Plio.js

```diff
@@ -48,13 +52,57 @@ export default {
     };
     return apiClient().post(process.env.VUE_APP_BACKEND_PLIOS, newPlioData);
   },
-  updatePlio(value, plioId) {
-    // creates a new draft plio
-    var plioData = {
-      value: value,
-      plio_id: plioId,
-      bucket: "plio-test-data",
-    };
-    return apiClient().post(process.env.VUE_APP_BACKEND_CREATE_PLIO, plioData);
+
+  async updatePlio(value, plioId) {
+    // handle items and questions being updated
+    value.items.forEach((item) => {
+      // pass a deepclone and not the reference
+      var cloneDeep = require("lodash.clonedeep");
+      var itemClone = cloneDeep(item);
+      ItemService.updateItem(itemClone);
+      QuestionService.updateQuestion(itemClone.details);
+    });
+
+    // prepare the payload for plio
+    var plioObject = {};
+
+    // handle (plio_title, status, created_by) being updated
+    plioObject.name = value.plio_title;
+    plioObject.status = value.status;
+    plioObject.created_by = 1; // TODO - HARDCODED FOR NOW
```

Comment:
let's hard code created_by in vue files only so that we don't have to touch JS files later.